cmd/watch-termination/main.go

```diff
@@ -78,6 +82,24 @@ func run() int {
 			client = kubernetes.NewForConfigOrDie(cfg)
 		}
 	}
+	var lock *flock.FLock
+
+	if len(*lockFile) > 0 {
+		tryLockTimeout := 30 * time.Second
+		if lockTimeout != nil {
```

Comment:
I think it cannot be nil since you are setting a default value of 30*time.Second